app/Enums/Decoder.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Enums;
+
+enum Decoder: string
+{
+    //
+    case CPU = 'cpu';
+    case NVIDIA_CUDA = 'nvidia_cuda';
+
+    public function getInitialParameters(): array
+    {
+        return config(sprintf('decoder.%s', $this->value)) ?? [];
```

Comment:
```suggestion
        return config(sprintf('decoder.%s', $this->value), []);
```

try using the default instead

Reply:
ok